sanitize.go

```diff
@@ -0,0 +1,143 @@
+package dns
+
+import "strings"
+
+// Dedup removes identical RRs from rrs. It preserves the original ordering.
+// The lowest TTL of any duplicates is used in the remaining one.
+//
+// TODO(miek): This function will be extended to also look for CNAMEs and DNAMEs.
```

Comment:
Yeah. Have the same feeling...it is also expensive to check for these.
On 26 Aug 2015 13:03, "Alex Sergeyev" <notifications@github.com> wrote:

> In sanitize.go
> <https://github.com/miekg/dns/pull/255#discussion_r37973616>:
>
> > @@ -0,0 +1,143 @@
> > +package dns
> > +
> > +import "strings"
> > +
> > +// Dedup removes identical RRs from rrs. It preserves the original ordering.
> > +// The lowest TTL of any duplicates is used in the remaining one.
> > +//
> > +// TODO(miek): This function will be extended to also look for CNAMEs and DNAMEs.
>
> I wonder if this should be optional... or in another function now... Not
> sure where this would be used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/miekg/dns/pull/255/files#r37973616>.
>